plugins/markdown_plugin.js

```diff
@@ -0,0 +1,19 @@
+import path from 'path'
+import fs from 'fs';
+
+const markdownPlugin = () => {
+  return {
+    name: 'our-custom-markdown-plugin',
+    load(id) {
+      if (id.endsWith('.md')) {
+        console.log("Importing markdown", id)
+        const filePath = path.resolve(id);
+        const content = fs.readFileSync(filePath, 'utf-8');
+        return `export default ${JSON.stringify(content)};`;
```

Comment:
@mohamed7genius On this level is where we would want to segment the contents according to a set of delimiters long term

Reply:
Do you mean by using https://github.com/al-mabsut/muslimah/pull/9/files#diff-790592b1ca541e03c2a2f16d009c838873ea3fcfdebe3d7a6a27f71e0f909b35R29 `mdStrToObj` ?